### PR TITLE
get_state_display will work with FSMIntegerField

### PR DESCRIPTION
**Pull request by hggh**
get_state_display has to convert the string into a integer before
access the dict of the fieldchoices

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -5,6 +5,8 @@ testpaths = tests
 markers =
     ignore_article: Configure the settings DJANGO_FSM_LOG_IGNORED_MODELS to ignore
                     Article Model.
+    ignore_article_integer: Configure the settings DJANGO_FSM_LOG_IGNORED_MODELS to ignore
```
Comment (ticosax) [inline setup.cfg]: What are you trying to achieve with this marker ?

Comment (hggh) [reply setup.cfg]: I have seen that the other model uses a marker. I simply copied it. Should I remove it?

Comment (ticosax) [reply setup.cfg]: Usually markers are used to select a subset of tests or to trigger side effects on others.
You don't do any of this here, so yes you can remove it.

**Comment by hggh** [comment]
@ticosax I have updated my PR and removed the marker stuff

**Pull request merged**